### PR TITLE
Make more tests report coverage to codecov.io

### PR DESCRIPTION
**Pull request by wangkuiyi**

### CODE REVIEW

File: scripts/test/mysql.sh [hunk 1]
```diff
@@ -34,10 +34,5 @@ python -c "import sqlflow_submitter.db"
 
 go generate ./...
 go install ./...
-
-# -p 1 is necessary since tests in different packages are sharing the same database
-# ref: https://stackoverflow.com/a/23840896
-# set test timeout to 900s since travis CI may be slow to run the case TestParse
-gotest -v -p 1 -timeout 900s ./...  -covermode=count -coverprofile=coverage.txt
-
+gotest -covermode=count -coverprofile=coverage.txt -timeout 900s  -v ./...
```
Comment (Yancey0623) [inline scripts/test/mysql.sh]: Some test cases saved the trained model into the same table as `sqlflow_models.my_dnn_modeo` c.f https://github.com/sql-machine-learning/sqlflow/search?l=Go&q=sqlflow_models.my_dnn_model

I'm afraid that parallel executing these test cases would overlap and cause some uncertain errors. And the title of this PR is `report coverage`, do you want to back to `-p 1` and do more tests in another PR?

Comment (wangkuiyi) [reply scripts/test/mysql.sh]: Let me revert to `-p 1` in this PR.

**Review by Yancey0623 — APPROVED** [review]
LGTM
